lib/timer_page.dart

```diff
@@ -220,6 +214,6 @@ class HundredsState extends State<Hundreds> {
   @override
   Widget build(BuildContext context) {
     String minutesStr = (hundreds % 100).toString().padLeft(2, '0');
-    return new Text(minutesStr, style: timerTextStyle, textAlign: TextAlign.start);
```

Comment:
Using `TextAlign.start`, `TextAlign.end` made the text non-horizontally centred, so I removed it.